package-lock.json

```diff
@@ -63,7 +60,11 @@
         "babel-external-helpers": "bin/babel-external-helpers.js"
       },
       "optionalDependencies": {
-        "@nicolo-ribaudo/chokidar-2": "2.1.8-no-fsevents"
+        "@nicolo-ribaudo/chokidar-2": "2.1.8-no-fsevents",
```

Comment:
Don't think this affects anything really but any idea why all these "funding urls" were updated?